build.sbt

```diff
@@ -187,6 +187,9 @@ val mimaFilterSettings = Seq {
     ProblemFilters.exclude[MissingClassProblem]("scala.collection.immutable.HashMap$HashMapKeys"),
     ProblemFilters.exclude[MissingClassProblem]("scala.collection.immutable.HashMap$HashMapValues"),
 
+    ProblemFilters.exclude[IncompatibleResultTypeProblem]("scala.collection.immutable.HashSet.--"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.collection.immutable.HashSet.--"),
```

Comment:
This won't fly, unfortunately. 

```
➜ scalac-pr 8684 -d /tmp <(echo 'object Main extends App { val v = collection.immutable.HashSet(""); v -- v }')
➜ scala-pr 8684 -cp /tmp Main
➜ scala-launch 2.12.10 -cp /tmp Main
java.lang.NoSuchMethodError: 'scala.collection.immutable.HashSet scala.collection.immutable.HashSet.$minus$minus(scala.collection.GenTraversableOnce)'
	at Main$.delayedEndpoint$Main$1(12:1)
	at Main$delayedInit$body.apply(12:1)
	at scala.Function0.apply$mcV$sp(Function0.scala:39)
```

The workaround is to add a type-case into the inherited implemnentation.

Reply:
modified to use a case in `Subtractable`